net-p2p/ipfs-desktop-bin/ipfs-desktop-bin-0.39.0.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit desktop xdg unpacker optfeature
+DESCRIPTION="A desktop client for IPFS. The IPFS's Native Application"
+HOMEPAGE="https://github.com/ipfs-shipyard/ipfs-desktop"
+SRC_URI="https://github.com/ipfs/ipfs-desktop/releases/download/v${PV}/ipfs-desktop-${PV}-linux-amd64.deb"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+x11-libs/gtk+:3
+x11-libs/libnotify
+dev-libs/nss
+x11-libs/libXScrnSaver
+x11-libs/libXtst
+x11-misc/xdg-utils
+app-accessibility/at-spi2-core
+sys-apps/util-linux
+app-crypt/libsecret
+"
```

Comment:
Those are not build-time dependencies. You only need them as RDEPEND.
There are also many missing runtime dependencies. You can check the linked libraries using `lddtree /opt/IPFS\ Desktop/ipfs-desktop`.

> dev-libs/expat
> dev-libs/glib:2
> dev-libs/nspr
> media-libs/alsa-lib
> media-libs/mesa
> net-print/cups
> sys-apps/dbus
> x11-libs/cairo
> x11-libs/gdk-pixbuf:2
> x11-libs/libX11
> x11-libs/libXcomposite
> x11-libs/libXdamage
> x11-libs/libXext
> x11-libs/libXfixes
> x11-libs/libXrandr
> x11-libs/libdrm
> x11-libs/libxcb:=
> x11-libs/libxkbcommon
> x11-libs/pango

Reply:
OK got it

Reply:
But I think the packages in list are commonly for desktop profile users..., =_=